src/components/NavigationReserve.js

```diff
@@ -0,0 +1,122 @@
+                strokeWidth="1.5"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+              <path d="M12 15C13.6569 15 15 13.6569 15 12C15 10.3431 13.6569 9 12 9C10.3431 9 9 10.3431 9 12C9 13.6569 10.3431 15 12 15Z" stroke="currentColor" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round" />
+            </svg>
+            <p className="font-bold leading-4 text-black border-b border-transparent hover:border-doctor focus:border-white ">MY RESERVATIONS</p>
+          </button>
+          <button type="button" className="flex items-center justify-start w-full py-3 space-x-6 font-bold text-red-500 rounded focus:outline-none focus:text-700 hover:text-red-600 ">
+            <svg width={24} height={24} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M14 8V6C14 5.46957 13.7893 4.96086 13.4142 4.58579C13.0391 4.21071 12.5304 4 12 4H5C4.46957 4 3.96086 4.21071 3.58579 4.58579C3.21071 4.96086 3 5.46957 3 6V18C3 18.5304 3.21071 19.0391 3.58579 19.4142C3.96086 19.7893 4.46957 20 5 20H12C12.5304 20 13.0391 19.7893 13.4142 19.4142C13.7893 19.0391 14 18.5304 14 18V16" stroke="currentColor" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round" />
+              <path d="M7 12H21M21 12L18 9M21 12L18 15" stroke="currentColor" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round" />
+            </svg>
+            <p className="leading-4 border-b border-transparent text-danger">Logout</p>
+          </button>
+          <div className="flex flex-col items-start w-full p-4 mt-3 rounded shadow bg-doctor jusitfy-start">
+            <p className="text-base font-bold leading-none text-black">Booking Capacity</p>
+            <p className="mt-3 text-xs leading-4 text-black">
+              Total slots booked as of
+              {' '}
+              <br />
+              <div>
+                <p>
+                  {date}
+                </p>
+              </div>
+            </p>
+            <div className="flex justify-end items-end text-right w-full mt-2.5">
+              <p className="text-base leading-4 text-white">80%</p>
+            </div>
+            <div className="flex justify-start w-full items-start mt-2.5">
+              <div className="w-10/12 h-0.5 bg-white rounded-full" />
+              <div className="w-4/12 h-0.5 bg-orange-200  rounded-full" />
+            </div>
+          </div>
+        </div>
+        <div className="flex items-center justify-start pt-6 pb-8 mx-6 space-x-2 ">
+          <div>
+            <img src="https://i.ibb.co/54vKnF3/Ellipse-3.png" alt="avatar" />
+          </div>
+          <div className="flex flex-col items-start space-y-1 jusitfy-start">
+            <p className="text-base leading-4 text-white cursor-pointer">Test Patient</p>
+            <p className="text-xs leading-3 text-orange-200 cursor-pointer">test@gmail.com</p>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
What can be do e that'd make this come follow DRY and SOLID rules? If you need some help (I gave some hints above ) feel free to reach out